src/discord/censor.py

```diff
@@ -123,12 +129,24 @@ async def __censor(self, message: discord.Message):
 
         # Make sure pinging through @everyone, @here, or any role can not happen
         mention_perms = discord.AllowedMentions(everyone=False, users=True, roles=False)
-        await webhook.send(
-            content,
-            username=f"{author} (auto-censor)",
-            avatar_url=avatar,
-            allowed_mentions=mention_perms,
-        )
+        if reply:
+            reply_message = (
+                f"*Replying to {reply.cached_message.author.mention}*"
+                f"\n>[{reply.cached_message.content[:80]}...]({reply.cached_message.jump_url})\n"
```

Comment:
This is not guaranteed to work since `reply.cached_message` might not be in the cache at all. You can have a message reference with or without any reference to its inner contents.

You probably will need to do something like:
```python
channel = bot.get_channel(message.reference.channel_id)
msg = await channel.fetch_message(message.reference.message_id)
```
to get the initial Message then read the contents.

On another note, the appending of this prefix will error in cases where the message reply is a very large length (1905 will guarantee the issue but it can happen with shorter replies as long as `len(reply message prefix) + len(user reply message) > 2000`).

Reply:
Could you take a look at the new commit? Thanks!!